pkg/rbac/directoryrolebinding/controller.go

```diff
@@ -6,9 +6,10 @@ import (
 	"time"
 
 	kitlog "github.com/go-kit/kit/log"
+	"github.com/pkg/errors"
 
 	rbacv1 "k8s.io/api/rbac/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
I thought this would have been an error importing two packages with the same name. Interesting.

Reply:
Why would it? You've aliased the packages to avoid that, right?

Reply:
Oh no my bad. I thought 'pkg/errors' was already in there 🤦‍♂so was wondering how it ever worked before you aliased it.